### ISSUE TITLE
Add Monsido to designsystemet.no

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#1425]
In addition to visitor numbers and statistics, Monsido can help us monitor how we comply with the requirements for universal design, that we use clear and correct language, that links work, that we find and correct typos, violations of language rules, that pages are updated ++

**Comment by mrosvik** [comment]
For bruker og tilgang, ta kontakt med Anne Tøsse Gjerløw i kommunikasjonsavdelinga.

**Comment by mrosvik** [comment]
Kan ikke bruke Monsido likevel da de har blitt kjøpt opp.
